src/test/java/org/folio/rest/api/TestBase.java

```diff
@@ -204,4 +206,12 @@ void assertGetNotFound(URL url, String tenantId) {
     assertThat(response.getStatusCode(), is(HttpURLConnection.HTTP_NOT_FOUND));
   }
 
+  public static JsonObject pojo2JsonObject(Object entity) {
+    try {
+      return PostgresClient.pojo2JsonObject(entity);
```

Comment:
@julianladisch could you please add details why this method should be used instead of JsonObject.mapFrom()?